docs/articles/documentation/reference/testcafe-api/runner/run.md

```diff
@@ -26,10 +26,12 @@ Parameter         | Type    | Description
 `quarantineMode`  | Boolean | Defines whether to enable the [quarantine mode](../../../guides/basic-guides/run-tests.md#quarantine-mode).                                                                                                                    | `false`
 `debugMode`       | Boolean | Specifies if tests run in the debug mode. If this option is enabled, test execution is paused before the first action or assertion so that you can invoke the developer tools and debug. In the debug mode, you can execute the test step-by-step to reproduce its incorrect behavior. You can also use the **Unlock page** switch in the footer to unlock the tested page and interact with its elements. | `false`
```

Comment:
```suggestion
`debugMode`       | Boolean | Specifies if tests run in the debug mode. If this option is enabled, test execution is paused before the first action or assertion, so that you can invoke the developer tools and debug. In the debug mode, you can execute the test step-by-step to reproduce its incorrect behavior. You can also use the **Unlock page** switch in the footer to unlock the tested page and interact with its elements. | `false`
```